zabbix-mysql-dump

```diff
@@ -49,7 +54,7 @@
 # DEFAULT VALUES
 #
 # following will store the backup in a subdirectory of the current directory
-DUMPDIR="$(dirname "$(realpath -s "$0")")"
```

Comment:
The  `realpath` programa is not installed by default in Ubuntu 12.04 nor 14.04.

I think a mention to this requirement is made necessary, or maybe another solution.